blog(embertimes134): ember-changeset 3.0.0 (#520)
ember-learn/ember-blog: source/2020-02-07-the-ember-times-issue-134.md

---

## [ember-changeset 3.0.0 Release 🎉](https://twitter.com/puekey/status/1224340431644708864)

> The [ember-changeset](https://github.com/poteto/ember-changeset) addon is officially released [version 3.0.0](https://github.com/poteto/ember-changeset/blob/master/CHANGELOG.md#300-2020-02-02). The addon provides a Changeset class which represents a set of valid changes to be applied onto any Object (`Ember.Object`, `DS.Model`, POJOs, etc). The idea is only the changeset's internal values are updated until we invoke the save method which would set and save the valid changes onto the provided object.

Reviewer: ```suggestion
The [ember-changeset](https://github.com/poteto/ember-changeset) addon officially released [version 3.0.0](https://github.com/poteto/ember-changeset/blob/master/CHANGELOG.md#300-2020-02-02). The addon provides a Changeset class that represents a set of valid changes for any Object (`Ember.Object`, `DS.Model`, POJOs, etc). The idea is only the changeset's internal values are updated until we invoke the save method which would set and save the valid changes onto the provided object.
```
